package.json

```diff
@@ -49,7 +50,8 @@
       {
         "command": "onevscode.circle-tracer",
         "title": "circle tracer",
-        "category": "ONE"
+        "category": "ONE",
+        "icon" : "$(play-circle)"
```

Comment:
![image](https://user-images.githubusercontent.com/17171963/151299735-d80e7137-884a-4f72-b59c-7dc99dfb96b6.png)

By hovering upon the icon, the title text "circle tracer" appears 
By clicking, one circle tracer window is opened.

Reply:
TODO: open the circle file at once